.github/workflows/release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build Container on Release
+
+on:
+
+  # Publish packages on release
+  release:
+    types: [published] 
+ 
+jobs:
+  build:
+    permissions:
+      packages: write
+
+    # Note this inherits the arguments for dependencies from the base container
+    # If you move this release trigger to the base-containers workflow you
+    # can tweak this, however the release will take longer.
+    runs-on: ubuntu-latest
```

Comment:
No runs-on is the GitHub action runner.

Reply:
The note says that since we are using the base container (that does have a specific ubuntu, vs. starting from scratch which you said was too slow) it will inherit that version. If there are eventually multiple base containers or versions you'd just change this to be a matrix with those versions.